skimage/segmentation/tests/test_clear_border.py

```diff
@@ -147,7 +147,7 @@ def test_clear_border_non_binary_out():
     image = np.array(
         [[1, 2, 3, 1, 2], [3, 3, 5, 4, 2], [3, 4, 5, 4, 2], [3, 3, 2, 1, 2]]
     )
-    out = np.empty_like(image)
+    out = image.copy()
```

Comment:
This seems to be a legit issue in this function. `out` needs to be initialized as a copy of `image`, for some reason `empty_like` returned a previous chunk of memory which had similar values to `image`, thus it was passing before the memory allocator changes in CPython 3.13

Reply:
Uff, that's a good catch. Good reminder to never rely on the content of `np.empty`.